conda/base/constants.py

```diff
@@ -156,7 +156,7 @@
     CONDA_PACKAGE_EXTENSION_V1,
 )
 CONDA_PACKAGE_PARTS = tuple(f"{ext}.part" for ext in CONDA_PACKAGE_EXTENSIONS)
-CONDA_TARBALL_EXTENSION = CONDA_PACKAGE_EXTENSION_V1  # legacy support for conda-build; remove this line  # NOQA
+CONDA_TARBALL_EXTENSION = CONDA_PACKAGE_EXTENSION_V1  # legacy support for conda-build
```

Comment:
why do we lose this comment?

Reply:
Some similar comments exist so that linters don't remove lines that look like they do nothing, but are used to re-export a symbol.